allure-adapter-plugin/src/main/kotlin/io/qameta/allure/gradle/adapter/AllureAdapterExtension.kt

```diff
@@ -170,7 +170,7 @@ open class AllureAdapterExtension @Inject constructor(
             "taskName" to taskName,
             "buildName" to project.name,
             "projectPath" to project.path,
-            "projectVersion" to project.version
+            "projectVersion" to project.version.toString()
```

Comment:
```suggestion
            // to string is to avoid unexpected interaction of custom objects and Json serializer
            "projectVersion" to project.version.toString()
```